<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelPicker</class>
 <widget class="QWidget" name="LevelPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="nameLabel">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: bold 50px;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>CIRCUIT CRAFT</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="levelsLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: bold 32px;
	color: rgb(50, 50, 50)
}</string>
       </property>
       <property name="text">
        <string>LEVELS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
	border: none;
}

QPushButton {
    border: 5px solid #707070;
    border-radius: 10px;
    background-color: #E0E0E0;
    color: black;
    font: bold 50px;
    padding: 5px; 
}

QPushButton:hover {
    background-color: #D8D8D8;
}

QPushButton:pressed {
    background-color: #CACACA;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #72aee0;
    border-radius: 10px;
    background-color: #afdaff;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #cfe9ff;
}

QPushButton:pressed {
    background-color: #acd8fc;
}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_4">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { border: 5px solid #88cf61; border-radius: 10px; background-color: #b8fe92; color: black; font: bold 48px; padding: 10px; position: fixed; } QPushButton:hover { background-color: #e8ffdb; } QPushButton:pressed { background-color: #c6fca9; }</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_7">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #938ae3;
    border-radius: 10px;
    background-color: #b7afff;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #dcd9ff;
}

QPushButton:pressed {
    background-color: #bbb5f7;
}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_8">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #e88ed1;
    border-radius: 10px;
    background-color: #ffafeb;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #fae1f3;
}

QPushButton:pressed {
    background-color: #f5bce6;
}</string>
          </property>
          <property name="text">
           <string>?</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #e0854c;
    border-radius: 10px;
    background-color: #febb92;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #fadecd;
}

QPushButton:pressed {
    background-color: #f2b894;
}</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #5e68d1;
    border-radius: 10px;
    background-color: #808af2;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #cacefa;
}

QPushButton:pressed {
    background-color: #abb2ff;
}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="button">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #de5758;
    border-radius: 10px;
    background-color: #fe9293;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #fccfcf;
}

QPushButton:pressed {
    background-color: #ff9c9c;
}</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
    border: 5px solid #e0c953;
    border-radius: 10px;
    background-color: #ffec8b;
    color: black;
    font: bold 48px;
    padding: 10px; 
position: fixed;
}

QPushButton:hover {
    background-color: #fcf4ca;
}

QPushButton:pressed {
    background-color: #f5e79f;
}</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
          <property name="class" stdset="0">
           <string>level_button</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="homeButton">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
    border: 5px solid #7398ff;
    border-radius: 10px;
    background-color: #a6bdff;
    color: black;
    font: bold 24px;
    padding: 10px; 
    min-width: 40px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #c7d5ff;
}

QPushButton:pressed {
    background-color: #a3baff;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
